app-frontend/src/app/core/services/project.service.js

```diff
@@ -189,6 +192,38 @@ export default (app) => {
         updateProject(params) {
             return this.Project.updateProject(params).$promise;
         }
+
+        getProjectLayerURL(project, token) {
+            let host = this.$location.host();
+            let protocol = this.$location.protocol();
+            let port = this.$location.port();
+            let formattedPort = port !== 80 && port !== 443 ? ':' + port : '';
+            let userId = 'rf_airflow-user';
+
+            let params = {
+                tag: new Date().getTime()
+            };
+
+            if (token) {
+                params.token = token;
+            }
+
+            let formattedParams = L.Util.getParamString(params);
+
+            return `${protocol}://${host}${formattedPort}` +
+                `/tiles/${project.organizationId}/${userId}` +
+                `/project/${project.id}/{z}/{x}/{y}/${formattedParams}`;
+        }
+
+        getProjectShareURL(project) {
+            let host = this.$location.host();
+            let protocol = this.$location.protocol();
+            let port = this.$location.port();
+            let formattedPort = port !== 80 && port !== 443 ? ':' + port : '';
+
+            return `${protocol}://${host}${formattedPort}` +
+                `/share/${project.id}`;
```

Comment:
I think this needs a hash (`#`) prefix; if I copy and paste the URL (rather than clicking the Preview button), I get redirected to `http://localhost:9091/share/bbaa6cbe-b3b2-4661-a2d8-e634eca2dee0#/browse/?minAcquisitionDatetime=2016-01-01T17:27:09.531Z&maxAcquisitionDatetime=2017-12-31T17:27:09.531Z&month=1&month=12&month=11&bbox=-136.75781250000003,-54.36775852406839,46.93359375000001,67.06743335108298`